conda_pypi/convert_tree.py

```diff
@@ -223,17 +203,22 @@ def convert_tree(
             else:  # more wheels for us to convert
                 channels = [local_channel]
 
-            solver = ReloadingLibMambaSolver(
+            solver = RattlerSolver(
                 str(prefix),
                 channels,
                 context.subdirs,
                 requested,
                 [],
+                # maybe need to add repodata_fn, command args
             )
 
-            with get_spinner(self._get_converting_spinner_message(channels)):
-                changes = self._convert_loop(
-                    max_attempts=max_attempts, solver=solver, tmp_path=tmp_path
-                )
-
-            return changes
+            try:
```

Comment:
I added this try statement as it seemed that there was panic error with Tokio.  I was hoping that by removing the solver that it would have to restart and subsequent solves could be performed.